addons/web/static/tests/views/fields/many2many_tags_avatar_field_tests.js

```diff
@@ -604,4 +609,39 @@ QUnit.module("Fields", (hooks) => {
             );
         }
     );
+
+    QUnit.test(
+        "Many2ManyTagsAvatarField: make sure that the arch context is passed to the form view call",
+        async function (assert) {
+            serverData.views = {
+                "partner,false,form": `<form><field name="display_name"/></form>`,
+            };
+
+            patchWithCleanup(browser, {
+                setTimeout: (fn) => fn(),
+            });
+
+            await makeView({
+                type: "list",
+                resModel: "turtle",
+                serverData,
+                arch: `<list editable="top"><field name="partner_ids" widget="many2many_tags_avatar" context="{ 'append_coucou': True }"/></list>`,
+                async mockRPC(route, args) {
+                    if (args.method === "onchange" && args.model === "partner") {
+                        if (args.kwargs.context.append_coucou) {
```

Comment:
Check the value it s maybe more robust ?

Reply:
Here you just check if append_coucou content a value but you don t check if it s the value is correct